test/specs/elements/Button/Button-test.js

```diff
@@ -116,6 +116,16 @@ describe('Button', () => {
       shallow(<Button label='hi' />)
         .should.have.className('labeled')
     })
+    it('adds the disabled className to the root element only when the disabled attribute is set', () => {
+      const wrapper = shallow(<Button label='foo' disabled />)
+      wrapper.should.have.className('disabled')
+      wrapper.children().forEach(child => child.should.not.have.className('disabled'))
+    })
+    it('adds the disabled className to the root element only when the disabled className is set', () => {
```

Comment:
This test is covered by `common.isConformant()` which asserts that all props and classNames are passed to the root element.  We can remove this one.